### PR TITLE
New Section for Development on Windows

### PR DESCRIPTION
**Pull request by DokuaAsiedu**
Created a note callout to address issue #9540: _make is not recognized as an internal or external command, operable program or batch file_

_Please check the following:_

-   [x] Do the tests still pass?[^1]

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

**Comment by DokuaAsiedu** [comment]
@lb- Could you have a look at this pr I made in reference to the [comment](https://github.com/wagtail/wagtail/issues/9540#issuecomment-1295522737) you left on issue #9540

File: docs/contributing/developing.md [hunk 1]
```diff
@@ -46,6 +46,18 @@ npm run build
 
 Any Wagtail sites you start up in this virtualenv will now run against this development instance of Wagtail. We recommend using the [Wagtail Bakery demo site](https://github.com/wagtail/bakerydemo/) as a basis for developing Wagtail. Keep in mind that the setup steps for a Wagtail site may include installing a release version of Wagtail, which will override the development version you've just set up. In this case, you should install the site before running the `pip install -e` step, or re-run that step after the site is installed.
 
+## Development on Windows
+Wagtail continues to update our documentation but development on Windows is not as documented as we would like it to be. We recommend setting up on a local virtual machine using our already available scripts, [docker-wagtail-develop](https://github.com/wagtail/docker-wagtail-develop) or [vagrant-wagtail-develop](https://github.com/wagtail/vagrant-wagtail-develop)
```
Comment (lb-) [inline docs/contributing/developing.md:50]:
```suggestion
Documentation for development on Windows has some gaps and should be considered a work in progress. We recommend setting up on a local virtual machine using our already available scripts, [docker-wagtail-develop](https://github.com/wagtail/docker-wagtail-develop) or [vagrant-wagtail-develop](https://github.com/wagtail/vagrant-wagtail-develop)
```

File: docs/contributing/developing.md [hunk 2]
```diff
@@ -46,6 +46,18 @@ npm run build
...
+
+If you are happy working on Windows, you will need to have the `make` utility installed in order to run commands associated with it. You can use Windows Powershell to install a package manager like [Chocolatey](https://chocolatey.org/install) and in turn use it install [make](https://chocolatey.org/install).
```
Comment (lb-) [inline docs/contributing/developing.md:52]:
```suggestion
If you are confident with Python and Node development on Windows and wish to proceed here are some helpful tips.

We recommend [Chocolatey](https://chocolatey.org/install) for managing packages in Windows. Once Chocolatey is installed you can then install the  [`make`](https://community.chocolatey.org/packages/make) utility in order to run common build and development commands.
```

Just trying to re-work this a bit to be less dense but also take you through discrete steps. Installing Chocolatey first and then make second.

File: docs/contributing/developing.md [hunk 3]
```diff
@@ -46,6 +46,18 @@ npm run build
...
+
+To effectively collaborate with other developers on different operating systems, we use CRLF to handle our line endings. You can configure this in Git using:
+
+```sh
```
Comment (lb-) [inline docs/contributing/developing.md:56]:
```suggestion
```doscon
```

As per https://docs.wagtail.org/en/stable/contributing/documentation_guidelines.html#when-using-console-terminal-code-blocks

Using `doscon` is our preferred approach for windows console/shell.

**Review by lb- — CHANGES_REQUESTED** [review]
Looking great. Just some recommendations for refinements. Thanks!

**Comment by DokuaAsiedu** [comment]
> Looking great. Just some recommendations for refinements. Thanks!

I've committed the suggestions you made. Could you please have a look at it again?

**Review by lb- — APPROVED** [review]
This is looking great, I will get this merged in shortly, thanks for taking on the feedback.

**Comment by DokuaAsiedu** [comment]
> This is looking great, I will get this merged in shortly, thanks for taking on the feedback.

Thanks for also giving the feedback 😊

**Pull request merged**